src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -304,6 +294,18 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
                     el.innerHTML = processedValue;
 
+                    if (domUtils.isBodyElement(el)) {
```

Comment:
Why not use `onBodyContentChanged` https://github.com/churkin/testcafe-hammerhead/blob/master/src/client/sandbox/code-instrumentation/properties/index.js#L330 event?